packages/services/api/src/modules/schema/module.graphql.ts

```diff
@@ -433,11 +433,11 @@ export default gql`
       )
     """
     The severity level of this schema change.
-    Note: A schema change with the impact \`SeverityLevelType.BREAKING\` can still be safe based on the usage (\`SchemaChange.isSafeBasedOnUsage\`).
+    Note: A schema change with the impact 'SeverityLevelType.BREAKING' can still be safe based on the usage ('SchemaChange.isSafeBasedOnUsage').
     """
     severityLevel: SeverityLevelType! @tag(name: "public")
     """
-    The reason for the schema changes severity level (\`SchemaChange.severityLevel\`)
+    The reason for the schema changes severity level ('SchemaChange.severityLevel')
```

Comment:
Just some cosmetic changes, it was driving me nuts that syntax highlighting in this file was broken.